codecov.yml

```diff
@@ -0,0 +1,51 @@
+# Setting coverage targets per flag
```

Comment:
will this file work with yaml extension? 😅

Reply:
let me see.